resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/LoggingPodStatusWatcher.scala

```diff
@@ -77,9 +87,18 @@ private[k8s] class LoggingPodStatusWatcherImpl(
     }
   }
 
+  override def reset(): Unit = {
```

Comment:
Shall we move this to line 69 (after `private def phase`) like the other branches?

Reply:
Done